docs/notebooks/image_to_image_arcface.md

```diff
@@ -0,0 +1,224 @@
+
+After fine-tuning, documents from each class should have similar embeddings, distinct from documents of other classes.
+<!-- #endregion -->
+
+<!-- #region id="M1sii3xdtD2y" -->
+## Data
+
+Our journey starts locally. We have to prepare the data and push it to the Jina AI Cloud and Finetuner will be able to get the dataset by its name. For this example,
+we already prepared the data, and we'll provide the names of training data (`stanford-cars-train`) directly to Finetuner.
+
+```{important} 
+We don't require you to push data to the Jina AI Cloud by yourself. Instead of a name, you can provide a `DocumentArray` and Finetuner will do the job for you.
+When working with documents where images are stored locally, please call `doc.load_uri_to_blob()` to reduce network transmission and speed up training.
+```
+<!-- #endregion -->
+
+```python id="L0NfPGbTkNsc"
+import finetuner
+from docarray import DocumentArray, Document
+
+finetuner.login(force=True)
+```
+
+```python id="ONpXDwFBsqQS"
+train_data = DocumentArray.pull('finetuner/stanford-cars-train', show_progress=True)
+query_data = DocumentArray.pull('finetuner/stanford-cars-query', show_progress=True)
+index_data = DocumentArray.pull('finetuner/stanford-cars-index', show_progress=True)
+
+train_data.summary()
+```
+
+<!-- #region id="mUoY1jq0klwk" -->
+## Backbone model
+Now let's see which backbone models we can use. You can see available models by calling `finetuner.describe_models()`.
+
+
+For this example, we're gonna go with `resnet50`.
+<!-- #endregion -->
+
+<!-- #region id="xA7IIhIOk0h0" -->
+## Fine-tuning
+
+Now that we have the training and evaluation datasets loaded as `DocumentArray`s and selected our model, we can start our fine-tuning run.
+<!-- #endregion -->
+
+```python id="qGrHfz-2kVC7"
+from finetuner.callback import EvaluationCallback
+
+run = finetuner.fit(
+    model='resnet50',
+    train_data='finetuner/stanford-cars-train',
+    batch_size=128,
+    epochs=5,
+    learning_rate=1e-3,
+    loss='ArcFaceLoss',
+    device='cuda',
+    sampler='random',
+    callbacks=[
+        EvaluationCallback(
+            query_data='finetuner/stanford-cars-query',
+            index_data='finetuner/stanford-cars-index',
+        )
+    ],
+)
+```
+
+<!-- #region id="9gvoWipMlG5P" -->
+Let's understand what this piece of code does:
+
+* As you can see, we have to provide the `model` which we picked before.
+* We also set `run_name` and `description`, which are optional,
+but recommended in order to retrieve your run easily and have some context about it.
+* Furthermore, we had to provide names of the `train_data`.
+* We set `ArcFaceLoss`.
+* Additionally, we use `finetuner.callback.EvaluationCallback` for evaluation.
+* Lastly, we set the number of `epochs` and provide a `learning_rate`.
```

Comment:
maybe explain a little bit on what is the content of `finetuner/stanford-cars-query` and `finetuner/stanford-cars-index`